### ISSUE TITLE
Unexpected Evaluation Network Results

### ISSUE THREAD

**Issue by linab8** [DeepLabCut/DeepLabCut#2439]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Bug description

Hello!
Due to some problems in Simba, I had to go back to DLC and add more videos to the training dataset. I've also corrected some of the old labels. However, after training the network, evaluation results show a larger test and train error compared to the old training set, even though I have almost double the frame number from the first training, and have corrected many labels so that they're more precise.
Do you have an idea why this might be happening? 

### Operating System

Windows 10 Pro 64bit

### DeepLabCut version

dlc version: 2.3.6.

### DeepLabCut mode

multi animal

### Device type

NVIDIA GeForce RTX 4090

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by n-poulsen** [comment]
Hi @linab8, how much larger is the test error?

Increasing the amount of data for the model does not necessarily mean evaluation results will be better, as your new images/videos might be more challenging (and if the videos look a bit different to the old ones, the model needs to "learn" how to predict poses in a wider range of images), however that does not mean your model isn't as good!

Did you:
- check that all of your labels are correct (plot them with `check_labels` and go through all images)?
- analyze a video to asses how well your model is doing on actual videos?

**Comment by linab8** [comment]
Hello @n-poulsen ! Here's the table with difference in test and train errors. 
![linagusteri error tablica](https://github.com/DeepLabCut/DeepLabCut/assets/148640900/2d359e17-47d0-413f-94f3-f03565d59198)

I understand that the larger amount of frames doesn't necessarily mean better evaluation results, however, I checked previous labels and corrected them so I expected better results. 
I didn't check labels on that original project, but I created a smaller project with some of the videos from the original on which I got similar test and train errors, and here I checked labels and they were all precise. I decided on this because of the large number of frames in the original project (>700 after retraining). Here are the errors from the smaller project:
![gusterirazlike errors](https://github.com/DeepLabCut/DeepLabCut/assets/148640900/75f4712f-f645-4b33-a8f6-955405c8c63e)

I analyzed the original model on videos and there were some ID swaps even after retraining, however, I hoped to correct those with interpolation in Simba. Then, when I started working in Simba, I had problems with outlier correction and I decided, together with an expert, that I should go back to DLC.
Do you think I've done something wrong in the procedure from what I've written here? My animals are around 140 pixels large, so these errors might not even be too large, however, the points are close to each other and based on this I viewed them as too large.

**Comment by linab8** [comment]
@n-poulsen  After my last comment I checked labels on the original videos and only 6 videos had problematic labels so I don't think this had impact.

**Comment by AlexEMG** [comment]
If you add novel data and make a new train/test split you cannot straightforwardly compare your results (as too many things are changing). If however, you add the new annotated images only to the training set and keep the test set the same, then you should indeed expect that the performance increases.

**Comment by linab8** [comment]
@AlexEMG Hello! I added new images only to the train set, the test set remained the same and includes all videos, sorry that I wasn't clear.

**Comment by n-poulsen** [comment]
@linab8 Did you correct the errors and re-train the model? Even just a few incorrect labels can lead to worse performance (from what I understand you did, but I just wanted to make sure). When looking at the numbers (model performance), I don't think there is anything wrong with your model (or with the procedure you used to train them).

There might be other things that you can try tweaking to improve performance such as the `pos_dist_thresh` (you can read more about it in the [pose_cfg.yaml Guideline Handbook](https://deeplabcut.github.io/DeepLabCut/docs/recipes/pose_cfg_file_breakdown.html)), but these parameters really depend on the data you're working with and we advise not to touch them to much as they work well for most datasets.

As this isn't really an issue with the code, the best place to discuss this would be the [forum](https://forum.image.sc/tag/deeplabcut) if you want more help from the DeepLabCut community!
